tests/legacy-cli/e2e/tests/build/rebuild-replacements.ts

```diff
@@ -19,7 +19,7 @@ export default async function() {
 
       // Should trigger a rebuild.
     await appendToFile('src/environments/environment.prod.ts', `console.log('PROD');`);
-    await waitForAnyProcessOutputToMatch(webpackGoodRegEx, 30000);
+    await waitForAnyProcessOutputToMatch(webpackGoodRegEx, 45000);
```

Comment:
Locally this rebuild takes 17s on latest, and 22s on nightly+TS3. 

I don't know if this is a regression in AOT or TS compilation but it 5s is ~1/4 of the total time so we should have a look after this PR.